validator/checks/url.py

```diff
@@ -20,7 +20,7 @@ class TextUrlExtractor(object):
     def __init__(self, **kwargs):
         pass
 
-    url_pattern = r'(?i)\b((?:https?://|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}/)(?:[^\s()\[\]<>]' \
+    url_pattern = r'(?i)\b((?:https?://|www\d{0,3}[.]|(!keepsafe://)[a-z0-9.\-]+[.][a-z]{2,4}/)(?:[^\s()\[\]<>]' \
```

Comment:
consider changing regexp so that decision if URL is being skipped or not is not hidden inside regexp. Regexp was there to extract all URLs in the first place. Now it plays another role